glide-core/src/client/value_conversion.rs

```diff
@@ -22,6 +22,8 @@ pub(crate) enum ExpectedReturnType {
     Lolwut,
     ArrayOfStringAndArrays,
     ArrayOfArraysOfDoubleOrNull,
+    ArrayOfMaps(&'a ExpectedReturnType<'a>),
```

Comment:
`rustc` required to use a reference, otherwise enum size is unlimited and then it required to define a lifetime specifier (`'a`)